src/components/FlowRunCreateForm.vue

```diff
@@ -21,20 +21,7 @@
             </template>
           </SchemaInputV2>
 
-          <template v-if="disableValidationCheckbox">
-            <p-tooltip>
-              <template #content>
-                <p>Parameters are always validated for deployments with parameter enforcement enabled.</p>
-                <p>You can disable this setting on <span><p-link :to="routes.deploymentEdit(deployment.id)">the deployment</p-link></span></p>
-              </template>
-              <div class="w-fit">
-                <p-checkbox v-model="shouldValidate" disabled label="Validate parameters before submitting" />
-              </div>
-            </p-tooltip>
-          </template>
-          <template v-else>
-            <p-checkbox v-model="shouldValidate" label="Validate parameters before submitting" />
-          </template>
+          <p-checkbox v-model="enforceParameterSchema" label="Enforce parameter schema" />
```

Comment:
I kind of like "Validate" better than "Enforce"; it implies there's a failure mode associated with it

```suggestion
          <p-checkbox v-model="enforceParameterSchema" label="Validate parameters" />
```